.travis.yml

```diff
@@ -146,8 +146,9 @@ jobs:
     - cargo run -p exonum-testkit --example timestamping
     - cargo run -p exonum-supervisor --example configuration_change
     - cargo run -p exonum-time --example simple_service
+    - cargo run -p exonum-keys --example key-generator -- -p passphrase -s a7839ea524f38d0e91a5ec96a723092719dc8a5b8a75f9131d9eb38f45e76344
 
-  # Integration tests.
+    # Integration tests.
```

Comment:
Nit: I don't think changing indent makes sense here; previously, the comment was aligned to the following list item mark (`- `).

Reply:
Yeah. It happened accidentally.